src/utils/user-role-utils.tsx

```diff
@@ -30,4 +43,13 @@ export default class UserRoleUtils {
 
     return UserRoleUtils.isAdmin() && adminPathname;
   };
+
+  /**
+   * Check if the logged-in user has devveloper role and is in admin route
+   *
+   * @returns boolean, indicates if user is developer
+   */
+  public static readonly developerMode = () => {
```

Comment:
You are not currently using this anywhere. If you want to follow the pattern as is done with AdminMode then you should be calling this in the home-screen and admin-screen rather than isDeveloper